scripts/update-bootstrap.ts

```diff
@@ -0,0 +1,50 @@
+import assert from 'assert';
+
+import cliPkg from '@crackle/cli/package.json';
+import bootstrapPkg from '@crackle-private/bootstrap/package.json';
+
+import { run, done, fromRoot } from './utils';
+
+const distTags = JSON.parse(
+  `${await run(`npm info @crackle/cli dist-tags --json`, {
+    encoding: 'utf-8',
+    stdio: 'pipe',
+  })}`,
+) as Record<string, string | undefined>;
+
+let version = distTags.bootstrap;
```

Comment:
You can specify the tag directly in the command if you want:

```suggestion
let version = `${await run(`npm info @crackle/cli dist-tags.bootstrap`, {
  encoding: 'utf-8',
  stdio: 'pipe',
})}`.trim();
```

Reply:
Sweet!